source/lmp/pair_deepmd.cpp

```diff
@@ -774,7 +782,7 @@ void PairDeepMD::compute(int eflag, int vflag) {
   if (!atom->sp_flag) {
     for (int ii = 0; ii < nall; ++ii) {
       for (int dd = 0; dd < 3; ++dd) {
-        f[ii][dd] += scale[1][1] * dforce[3 * ii + dd];
+        f[ii][dd] += conversion_factor * dforce[3 * ii + dd];
```

Comment:
I recommend to preserve scale, in case we need to revise it in the future.

Reply:
I see at the  virial we did not multiply the scale...

Reply:
I don't get what the scale is used for. But if the scale should be kept, a better way is something like `scale *= factor`.